libs/@guardian/source-react-components/src/choice-card/ChoiceCard.tsx

```diff
@@ -81,14 +92,50 @@ export const ChoiceCard = ({
 		return !!defaultChecked;
 	};
 
+	const transformProvidedTheme = (
+		providedTheme: typeof choiceCardThemeDefault.choiceCard | undefined,
+	): ChoiceCardTheme => {
+		if (!providedTheme) {
+			return {};
+		}
+		return {
+			textUnSelected: providedTheme.textLabel,
+			textSelected: providedTheme.textChecked,
+			borderUnSelected: providedTheme.border,
+			borderSelected: providedTheme.borderChecked,
+			backgroundSelected: providedTheme.backgroundChecked,
+			...providedTheme,
+		};
+	};
+
+	const getCombinedTheme = (
+		providedTheme: ChoiceCardTheme,
+	): ChoiceCardFullTheme => {
+		if (typeof theme !== 'string' && theme) {
+			return {
+				...choiceCardThemeLight,
+				...providedTheme,
+				...theme,
+			};
+		}
+		if (typeof theme === 'string') {
+			switch (theme) {
```

Comment:
see earlier comment - this feels wrong to have a separate switch between light and dark mode. are we not just wanting to give the option of passing a dark mode theme in?